### PR TITLE
making the bundle compatible with symfony PR6

### PR DESCRIPTION
**Pull request by diegoholiveira**
I make a few changes in the bundle to it be compatible with symfony PR6. :)

### CODE REVIEW

**Comment by lsmith77** [comment]
i realize that right now its kinda hard to follow whats happening, but thats actually a step back .. the Bundle is currently compatible with the latest development version. once Symfony2 reaches a stable state things will slow down and Bundles will start being versioned.

**Comment by diegoholiveira** [comment]
can you explain to me why this is a step back? :)

**Comment by lsmith77** [comment]
you provided a patch for making this bundle compatible with an older version of Symfony2 core.

**Comment by diegoholiveira** [comment]
I'm using the PR6. It's the last version available to download in symfony-reloaded.org. Are you using the development version? Because i believe that is the only version higher than the PR6.

**Comment by lsmith77** [comment]
https://github.com/FriendsOfSymfony/FacebookBundle/pull/14#issuecomment-816900
"the Bundle is currently compatible with the latest development version"

**Comment by diegoholiveira** [comment]
thanks. i will update my project to the development version. :)
